lib/opal/nodes/arglist.rb

```diff
@@ -45,7 +45,7 @@ def compile
           if code.empty?
             code = join
           else
-            code << fragment('.concat(') << join << fragment(')')
+            code << fragment('.concat([') << join << fragment('])')
```

Comment:
```js
["a"].concat("b") // ["a", "b"]
["a"].concat(["b"]) // ["a", "b"]
["a"].concat([["b"]]) // ["a", ["b"]]
```
JavaScript 😱

`push` is more predictable but the return value is the new array length (and also mutate the array).